docs/developer/install-latest-standalone-ui.md

```diff
@@ -0,0 +1,78 @@
+# Process to deploy latest standalone UI  synced with latest ui-backend and epinio
+
+### Step 1: get latest epinio update
+
+Open a terminal pointing to [epinio/epinio](https://github.com/epinio/epinio) repo
+Ensure submodules are synced. As help this can be executed:
+```
+git submodule sync --recursive && git submodule update --init --recursive && git submodule foreach --recursive "git checkout . && git reset --hard && git clean -dffx"
+```
+Execute:
+```
+export KUBECONFIG=$PWD/tmp/acceptance-kubeconfig
+```
+Ensure you have nodes running:
+
+```
+epinio git:(main) kubectl get nodes
+NAME                             STATUS   ROLES                  AGE    VERSION 
+k3d-epinio-acceptance-server-0   Ready    control-plane,master   109s   v1.20.11+k3s2 
+```
+Execute:
+```
+make install-cert-manager
+make prepare_environment_k3d  
+```
+Check epinio version with `epinio info`:
+It should output the current version, plus and aditional number
+
+```✔️  Epinio Environment 
+Platform: k3s 
+Kubernetes Version: v1.20.11+k3s2 
+Epinio Server Version: v0.8.0-124-g57f85060 
+Epinio Client Version: v0.8.0-124-g57f85060 
+```
+######Optional: create a symlink for the latest versions
+`ln -s ~/epinio/dist/epinio-linux-amd64 /usr/local/bin/epinio`
+
+---
+
+## To run the latest version of the UI
+
+#### Step 2: prepare ui-backend
+
+Open another terminal pointing to [epinio/ui-backend](https://github.com/epinio/ui-backend) repo
+Execute:
+
+```
+cp src/jetstream/config.example src/jetstream/config.properties 
+```
+
+Go to `/src/jetstream/` and ensure the following line is enabled in `config.properties` file:
+`AUTH_ENDPOINT_TYPE=epinio `
+
+Execute the following comand and copy the id displayed:
+`epinio settings show`
+
+Execute the following with the copied id:
+```
+go build && EPINIO_API_URL=https:${COPIED_ID}.omg.howdoi.website EPINIO_WSS_URL=https://epinio.${COPIED_ID}.omg.howdoi.website EPINIO_API_SKIP_SSL=true EPINIO_VERSION=dev ./jetstream  
```

Comment:
`EPINIO_API_URL=https:${COPIED_ID}.omg.howdoi.website`
`https:` only? typo or is it correct?